src/MagnumPlugins/BasisImporter/BasisImporter.cpp

```diff
@@ -717,7 +776,10 @@ template<UnsignedInt dimensions> Containers::Optional<ImageData<dimensions>> Bas
                     flipped = false;
                     break;
                 /* We'd be in the isUncompressed branch above for this */
-                case TargetFormat::RGBA8: CORRADE_INTERNAL_ASSERT_UNREACHABLE(); /* LCOV_EXCL_LINE */
+                case TargetFormat::RGBA8:
+                case TargetFormat::RGB16F:
+                case TargetFormat::RGBA16F:
+                    CORRADE_INTERNAL_ASSERT_UNREACHABLE(); /* LCOV_EXCL_LINE */
```

Comment:
Given that codecov complains here, it has to be this now

```suggestion
                /* LCOV_EXCL_START */
                case TargetFormat::RGBA8:
                case TargetFormat::RGB16F:
                case TargetFormat::RGBA16F:
                    CORRADE_INTERNAL_ASSERT_UNREACHABLE(); 
                /* LCOV_EXCL_STOP */
```